script/brave_license_helper.py

```diff
@@ -132,6 +132,21 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
             "URL": "https://github.com/brave/Sparkle",
             "License": "MIT",
         },
+        os.path.join('brave', 'third_party', 'cryptography'): {
+            "Name": "cryptography",
+            "URL": "https://cryptography.io",
+            "License": "Apache-2.0",
+            # Because of the way we install this third-party library, there is
+            # no LICENSE file in its directory. So use an appropriate file from
+            # another dependency.
+            "License File": \
+                "/brave/third_party/rust/either/v1/crate/LICENSE-APACHE",
```

Comment:
Use this one instead so that it won't be removed if the rust dependency is replaced later (or changes its license): https://github.com/brave/brave-core/blob/master/common/licenses/Apache-2.0

Reply:
Resolved in 06ce51a37ceb3b9a8d44a1b3b883b3f44055c5f5.